cmd/ci-secret-bootstrap/main.go

```diff
@@ -488,6 +633,14 @@ func main() {
 	if err != nil {
 		errs = append(errs, err)
 	}
+
+	if o.validateBWItemsUsage {
+		unusedItems := getUnusedBWItems(o.config, bwClient, sets.NewString(o.bwAllowItems.Strings()...))
+		if len(unusedItems) > 0 {
+			logrus.Fatal("unused Bitwarden items found.")
```

Comment:
Please append to `errs` instead, so we will only fail the job after syncing everything